Assets/MRTK/SDK/Editor/Inspectors/UX/ObjectManipulator/ObjectManipulatorInspector.cs

```diff
@@ -41,11 +43,23 @@ public class ObjectManipulatorInspector : UnityEditor.Editor
         private SerializedProperty onHoverEntered;
         private SerializedProperty onHoverExited;
 
+        private SerializedProperty elasticTypes;
+        private SerializedProperty translationElasticConfigurationObject;
+        private SerializedProperty rotationElasticConfigurationObject;
+        private SerializedProperty scaleElasticConfigurationObject;
+        private SerializedProperty translationElasticExtent;
+        private SerializedProperty rotationElasticExtent;
+        private SerializedProperty scaleElasticExtent;
+
         bool oneHandedFoldout = true;
         bool twoHandedFoldout = true;
         bool constraintsFoldout = true;
         bool physicsFoldout = true;
         bool smoothingFoldout = true;
+        bool elasticsFoldout = true;
+        bool translationElasticFoldout,
+             rotationElasticFoldout,
```

Comment:
Nit: have each bool have its own bool type prefix i.e. 

```c#
bool translationElasticFoldout = false;
bool rotationElasticFoldout = false;
```

The reasons being:
1. Having the default value explicitly on the line it's defined/closer to it makes it easier to see what the default value is.
2. Reduces the chances of merge issues/easier merges (this is more likelly to happen in much larger codebases and isn't as applicable in this case, but it's still a good practice to have to avoid problems in general.
3. Not a huge difference in verbosity (i.e. same line count, some more characters).
4. Consistent with existing stuff.

Reply:
The reason I did this was because they were semantically linked (in my head, at least). Fixed according to your suggestion now!